CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
-- Added an analysis report page (html and PDF format) containing phenotype, gene panels and variants that are relevant to solve a case.
```

Comment:
Why is this line not in the rebased version? This line comes from your earlier PR, right? Then it should be included if you have rebased this PR...

Reply:
It was included but I removed it at the conflict resolve. I'll put it back!